neo/Network/P2P/TaskManager.cs

```diff
@@ -107,7 +114,8 @@ private void OnRegister(VersionPayload version)
         private void OnRestartTasks(InvPayload payload)
         {
             knownHashes.ExceptWith(payload.Hashes);
-            globalTasks.ExceptWith(payload.Hashes);
+            foreach (UInt256 hash in payload.Hashes)
+                globalTasks.Remove(hash);
```

Comment:
`globalTasks[hash]--`?

Reply:
Shouldn’t restart clear them completely rather than subtract one?

Reply:
I guess I need to look over the way this is used more and see what makes most sense for it to be doing. Can you please explain the reasoning behind what this code did before this change to allow multiple concurrent tasks?